tests/functional/u/undefined/undefined_variable_typing.rc

```diff
@@ -1,2 +1,4 @@
 [testoptions]
 except_implementations=PyPy
+[VARIABLES]
+additional_builtins=__additional_builtin__
```

Comment:
@DanielNoord linting this file passes on the command line with giving ``--additional-builtins=__additional_builtin__`, but I had trouble getting a config file to read this option. Do you know why this is failing?